File: app/src/main/java/com/chess/cryptobot/model/response/HistoryResponse.java
package com.chess.cryptobot.model.response;

import java.time.LocalDateTime;

public interface HistoryResponse {

    LocalDateTime getHistoryTime();

    String getHistoryName();

    String getHistoryMarket();

    Double getHistoryAmount();

    Double getHistoryPrice();

    String getHistoryAction();

    Integer getProgress();
}

File: app/src/main/java/com/chess/cryptobot/model/response/HistoryResponse.kt
package com.chess.cryptobot.model.response

import java.time.ZonedDateTime

interface HistoryResponse {
    val historyTime: ZonedDateTime?
    val historyName: String?
    val historyMarket: String?
    val historyAmount: Double?
    val historyPrice: Double?
    val historyAction: String?
    val progress: Int?
}